### DISCUSSION TITLE
Curation report February 21

### DISCUSSION THREAD

**Discussion by andrawaag** [wikipathways/GPML-Repo#45] [Curation reports]
Hi All, 

   A bit overdue, but I went through the curation cycle. There were quite some new pathways of which many were of high quality. I approved all of those that had all items (sorry I lost count)
   
  * I did miss the delete pathways step this time. I don't know what I did wrong. 

Andra

**Comment by khanspers** [comment]
Thanks @andrawaag. The deletion task is in the Additional Tasks steps, but it is sort of buried in the text. I'm updating it now to make it more clear. 
Also, I just checked the list of pathways tagged for deletion and there's nothing to delete right now so no worries.
